Attempt to list all entries causes panic

``` go
$ ./bin/toodoo list
panic: open templates/todo.tmpl: no such file or directory

goroutine 16 [running]:
runtime.panic(0x1348c0, 0xc208024420)
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/panic.c:279 +0xf5
github.com/mikespokefire/toodoo/toodoo.(*TodoList).List(0xc20800e040)
    /Users/gavinlaking/Source/go/src/github.com/mikespokefire/toodoo/toodoo/todolist.go:44 +0xc8
main.list()
    /Users/gavinlaking/Source/go/src/github.com/mikespokefire/toodoo/toodoo.go:72 +0x47
main.main()
    /Users/gavinlaking/Source/go/src/github.com/mikespokefire/toodoo/toodoo.go:20 +0x447

goroutine 19 [finalizer wait]:
runtime.park(0x14c70, 0x239d20, 0x2390c9)
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0x239d20, 0x2390c9)
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/proc.c:1445

goroutine 17 [syscall]:
runtime.goexit()
    /usr/local/Cellar/go/1.3/libexec/src/pkg/runtime/proc.c:1445
```

--- a/toodoo/todolist.go
+++ b/toodoo/todolist.go
@@ -52,13 +52,11 @@ func newTodo(name string) *Todo {
 
 func (todos *TodoList) Read() {
 	file, err := ioutil.ReadFile(saveFileLocation())
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	json_err := json.Unmarshal(file, todos)
-	if json_err != nil {
-		log.Fatal(json_err)
+	if err == nil {
+		json_err := json.Unmarshal(file, todos)
+		if json_err != nil {
+			log.Fatal(json_err)
+		}
 	}
 }
 

